resources/js/src/app/shared/UrlBuilder.ts

```diff
@@ -16,12 +16,22 @@ class UrlBuilder {
 
         urlParams = urlParams.split('+').join(' ');
         while ((tokens = regex.exec(urlParams)) !== null) {
-            params[decodeURIComponent(tokens[1])] = decodeURIComponent(tokens[2]);
+            params[decodeURIComponent(tokens[1])] = this.sanitizeHTML(decodeURIComponent(tokens[2]));
         }
 
         return params;
     }
 
+    sanitizeHTML(str : string): string {
+
+        const map = {
```

Comment:
Extend map to handle additional characters, such as > (&gt;) and & (&amp;) please